client/landing/gutenboarding/onboarding-block/language/index.tsx

```diff
@@ -58,13 +61,11 @@ const LanguageStep: React.FunctionComponent = () => {
 				setFilter( languageGroup.id );
 			};
 			return (
-				<Button
-					key={ languageGroup.id }
-					className={ isSelected ? 'selected' : '' }
-					onClick={ onClick }
-				>
-					{ languageGroup.name( __ ) }
-				</Button>
+				<div key={ languageGroup.id }>
+					<Button onClick={ onClick } className={ 'language__language-group' }>
+						<span className={ isSelected ? 'selected' : '' }>{ languageGroup.name( __ ) }</span>
```

Comment:
Would be good to have `is-selected` format for modifier classes.
https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/css.md?term=css

Reply:
thanks I updated that 👍